src/main/java/uk/gov/hmcts/ccd/domain/model/search/elasticsearch/UICaseSearchResult.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.ccd.domain.model.search.elasticsearch;
+
+import lombok.Data;
+import lombok.NonNull;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.emptyList;
+
+@Data
+public class UICaseSearchResult {
```

Comment:
we are trying to follow the convention that all internal api result objects are denoted by `View`